### PR TITLE
fix(textwidget): Fix the TextWidget

### PR DESCRIPTION
**Pull request by DavidBerger98**
Part of the API of the widgets has changed in vtk.js making the widget crash.

### CODE REVIEW

**Comment by DavidBerger98** [comment]
Fixes #456

File: src/vtk/TextWidget/behavior.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import macro from '@kitware/vtk.js/macro';
 const MAX_POINTS = 1;
 
 export default function widgetBehavior(publicAPI, model) {
-  model.classHierarchy.push('vtkDistanceWidgetProp');
-  let isDragging = null;
+  model.classHierarchy.push('vtkLabelWidgetProp');
+  model._isDragging = false;
```
Comment (finetjul) [inline src/vtk/TextWidget/behavior.js:7]: I doubt you have to make isDragging a member variable. Do you confirm it does not work with `let isDragging = null` ?

File: src/vtk/TextWidget/behavior.js [hunk 2]
```diff
@@ -3,8 +3,8 @@ import macro from '@kitware/vtk.js/macro';
 const MAX_POINTS = 1;
 
 export default function widgetBehavior(publicAPI, model) {
-  model.classHierarchy.push('vtkDistanceWidgetProp');
-  let isDragging = null;
+  model.classHierarchy.push('vtkLabelWidgetProp');
```
Comment (finetjul) [inline src/vtk/TextWidget/behavior.js:6]: vtkTextWidgetProp instead

File: src/vtk/TextWidget/behavior.js (hunk 1 above)
Comment (DavidBerger98) [reply src/vtk/TextWidget/behavior.js:7]: It works, but in vtk.js the other widgets (see behavior.js in ShapeWidet, SphereWidget, SplineWidget, etc.) all have `model._isDragging`.
Shouldn't it be consistent with this?

**Comment by finetjul** [comment]
LGTM

**Review by finetjul — APPROVED** [review]

**Comment by floryst** [comment]
LGTM!

**Pull request merged**